include/utility

```diff
@@ -297,6 +297,8 @@ _GLIBCXX_BEGIN_NAMESPACE_VERSION
   // extract the elements in a tuple.
   template<size_t... _Indexes> struct _Index_tuple { };
 
+#if __GNUC__ >= 8 || defined(_GLIBCXX_USE_MAKE_INTEGER_SEQ)
```

Comment:
What is the `_GLIBCXX_USE_MAKE_INTEGER_SEQ` check for, to allow clang to use its built-in here?

Reply:
In upstream libstdc++, the later check for `_GLIBCXX_USE_MAKE_INTEGER_SEQ` is because it gets set when `__has_builtin(__make_integer_seq)` is true, to avoid doing that check again. But this check is before it gets set, so will never be true ... unless it's intended to allow clang users to manually define it on the command line to use the clang built-in ... but it seems like just `#if __GNUC__ >= 8 || defined __clang__` would be simpler and not require them to do anything manually.

Or it's a thinko and should just be `#if __GNUC__ >= 8` here, and only check the `_GLIBCXX_USE_MAKE_INTEGER_SEQ` macro below on line 355.

Reply:
This was a copy/paste from the original commit (1e245c1).

> Or it's a thinko and should just be `#if __GNUC__ >= 8` here, and only check the `_GLIBCXX_USE_MAKE_INTEGER_SEQ` macro below on line 355.

Done: commit 6a11d0697dedef039219d4fb9e0bc4293601c25b